docs/src/main/java/io/quarkus/docs/generation/ExtensionJson.java

```diff
@@ -0,0 +1,21 @@
+package io.quarkus.docs.generation;
+
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class ExtensionJson {
+    public List<Extension> extensions;
+
+    @JsonIgnoreProperties(ignoreUnknown = true)
+    public static class Extension {
+        public String name, groupId, artifactId;
+
+        public String getConfigName() {
+            if (name != null)
+                return name;
+            return "NAME MISSING: " + groupId + ":" + artifactId;
```

Comment:
Is it possible to log a warning and falling back to a temporary name as done in  https://github.com/quarkusio/quarkus/pull/4522 ?

Reply:
Sure, but I wanted to see which extensions were missing quickly.

Reply:
Fixed.